pennylane/devices/default_mixed.py

```diff
@@ -494,6 +508,22 @@ def _apply_operation(self, operation):
             self._apply_channel(matrices, wires)
 
     # pylint: disable=arguments-differ
+
+    def statistics(self, observables, shot_range=None, bin_size=None):
+        """Applies a readout error to the measurement outcomes of any observable if
+        readout_prob is non-zero. It applies to the following properties being
+        measured: Expectation, Variance, Sample, and Probability. This is done
+        by applying a BitFlip channel to the wires being measured after applying the
+        circuit rotations to the pre-rotated state. The results of state and density_matrix
+        are unaffected by the readout error because they correspond to the pre-rotated state."""
```

Comment:
The closing `"""` come in a new line (see e.g., docstrings in `QubitDevice` class).